contracts/TopShot.cdc

```diff
@@ -1717,22 +1725,34 @@ access(all) contract TopShot: NonFungibleToken {
                         "instagram": MetadataViews.ExternalURL("https://www.instagram.com/nbatopshot")
                     }
                 )
-                case Type<MetadataViews.Royalties>():
-                    let royaltyReceiver: Capability<&{FungibleToken.Receiver}> =
-                        getAccount(TopShot.RoyaltyAddress()).capabilities.get<&{FungibleToken.Receiver}>(MetadataViews.getRoyaltyReceiverPublicPath())!
-                    return MetadataViews.Royalties(
-                        [
-                            MetadataViews.Royalty(
-                                receiver: royaltyReceiver,
-                                cut: 0.05,
-                                description: "NBATopShot marketplace royalty"
-                            )
-                        ]
-                    )
+            case Type<MetadataViews.Royalties>():
+                let royaltyReceiver: Capability<&{FungibleToken.Receiver}> =
+                    getAccount(TopShot.RoyaltyAddress()).capabilities.get<&{FungibleToken.Receiver}>(MetadataViews.getRoyaltyReceiverPublicPath())!
+                return MetadataViews.Royalties(
+                    [
+                        MetadataViews.Royalty(
+                            receiver: royaltyReceiver,
+                            cut: 0.05,
+                            description: "NBATopShot marketplace royalty"
+                        )
+                    ]
+                )
+            // case Type<MetadataViews.CrossVMPointer>():
+            //     return self.resolveCrossVMPointerView()
         }
         return nil
     }
 
+    // resolveCrossVMPointerView resolves the CrossVMPointer view
+    // access(all) view fun resolveCrossVMPointerView(): MetadataViews.CrossVMPointer {
```

Comment:
Commented out until [this PR](https://github.com/onflow/flow-nft/pull/243/files#diff-c7956c28b08398abbebc13e67db6c3502f28e39764c642bef07f95b19ee95d4cR22)'s changes are merged and pushed on-chain - this new metadata view should be returned both from contract and nft metadata views; this should provide the custom contract FlowEVM address